usaspending_api/download/download_utils.py

```diff
@@ -77,7 +77,17 @@ def log_new_download_job(request, download_job):
 
 
 def construct_data_date_range(provided_filters: dict) -> str:
-    string = f"FY{provided_filters.get('fy')}Q1"
-    if provided_filters.get("quarter") != 1:
-        string += f"-Q{provided_filters.get('quarter')}"
+    if provided_filters.get("fy") is not None and provided_filters.get("fy") > 2019:
+        if provided_filters.get("fy") == 2020:
+            string = f"FY{provided_filters.get('fy')}Q1"
+        else:
+            string = f"FY{provided_filters.get('fy')}P01"
+        if provided_filters.get("period") is not None and provided_filters.get("period") != 1:
```

Comment:
To avoid confusion I would either remove the check for `period != 1` or add in another check to handle the case of period == 1. The current validation won't allow a period of 1 with the `/download/accounts` endpoint but if a period of 1 did hit this logic it would come back as `FY2021P01-QNone`.

Reply:
That's fair